PowerEditor/src/WinControls/TabBar/TabBar.cpp

```diff
@@ -634,7 +634,13 @@ LRESULT TabBarPlus::runProc(HWND hwnd, UINT Message, WPARAM wParam, LPARAM lPara
 		{
 			if (_mightBeDragging && !_isDragging)
 			{
-				if (++_dragCount > 2)
+				// Grrr! Who has stolen focus and eaten the WM_LBUTTONUP?!
+				if (GetKeyState(VK_LBUTTON) >= 0)

```

Comment:
##
[MSDN](https://msdn.microsoft.com/en-us/library/windows/desktop/ms646301(v=vs.85).aspx) says:
Return value

Type: SHORT

The return value specifies the status of the specified virtual key, as follows:
    

- If the high-order bit is 1, the key is down; otherwise, it is up.
- If the low-order bit is 1, the key is toggled. A key, such as the CAPS LOCK key, is toggled if it is turned on. The key is off and untoggled if the low-order bit is 0. A toggle key's indicator light (if any) on the keyboard will be on when the key is toggled, and off when the key is untoggled.

##

So I feel, high-order bit should be checked to determine. Key is pressed or not.

Reply:
The return is a signed short. ON and Toggle ON will both be negative values, while OFF & Toggle OFF is 0 and 1 respectively